dynawo/3rdParty/CMakeLists.txt

```diff
@@ -0,0 +1,617 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```

Comment:
Would it be possible to split this file (1 file for each 3rd party?)

Reply:
I think yes but I would prefer do it in another commit to avoid problems as it is working